docs/content/integrations/embedded-elt.mdx

```diff
@@ -5,167 +5,253 @@ description: Lightweight ELT framework for building ELT pipelines with Dagster,
+    "TARGET": "MY_SF",
+    "defaults": {"mode": "full-refresh", "object": "{stream_schema}_{stream_table}"},
+    "streams": {
+        "public.accounts": None,
+        "public.users": None,
+        "public.finance_departments": {"object": "departments"},
+    },
+}
+
+
+@sling_assets(replication_config=replication_config)
+def my_assets(context, sling: SlingResource):
+    yield from sling.replicate(
+        replication_config=replication_config,
+        dagster_sling_translator=DagsterSlingTranslator(),
+    )
 ```
 
 Similarily, you can define file/storage connections:
 
 ```python startafter=start_storage_config endbefore=end_storage_config file=/integrations/embedded_elt/s3_snowflake.py
-source = SlingSourceConnection(
+source = SlingConnectionResource(
+    name="MY_S3",
     type="s3",
     bucket="sling-bucket",
-    access_key_id=os.getenv("AWS_ACCESS_KEY_ID"),
-    secret_access_key=os.getenv("AWS_SECRET_ACCESS_KEY"),
+    access_key_id=EnvVar("AWS_ACCESS_KEY_ID"),
+    secret_access_key=EnvVar("AWS_SECRET_ACCESS_KEY"),
 )
 
-sling = SlingResource(source_connection=source, target_connection=target)
-
-asset_def = build_sling_asset(
-    asset_spec=AssetSpec("my_asset_name"),
-    source_stream="s3://my-bucket/my_file.parquet",
-    target_object="marts.my_table",
-    primary_key="id",
-)
+sling = SlingResource(connections=[source, target])
+
+replication_config = {
+    "SOURCE": "MY_S3",
+    "TARGET": "MY_SF",
+    "defaults": {"mode": "full-refresh", "object": "{stream_schema}_{stream_table}"},
+    "streams": {
+        "s3://my-bucket/my_file.parquet": {
+            "object": "marts.my_table",
+            "primary_key": "id",
+        },
+    },
+}
+
+
+@sling_assets(replication_config=replication_config)
+def my_assets(context, sling: SlingResource):
+    yield from sling.replicate(
+        replication_config=replication_config,
+        dagster_sling_translator=DagsterSlingTranslator(),
+    )
 ```
+
+## Relevant APIs
+
+| Name                                                                              | Description                                                                    |
+| --------------------------------------------------------------------------------- | ------------------------------------------------------------------------------ |
+| <PyObject module="dagster_embedded_elt.sling" object="sling_assets" />            | The core Sling asset factory for building syncs                                |
+| <PyObject module="dagster_embedded_elt.sling" object="SlingResource" />           | The Sling Resource used for handing credentials to databases and object stores |
+| <PyObject module="dagster_embedded_elt.sling" object="DagsterSlingTranslator" />  | A translator for specifying how to map between Sling and Dagster types         |
+| <PyObject module="dagster_embedded_elt.sling" object="SlingConnectionResource" /> | A Sling connection resource for specifying the connection details              |
```

Comment:
> the connection details

For what? I suggest including `resource` in here somewhere to make it more specific

Reply:
updated to: "A Sling connection resource for specifying database and storage connection credentials" how's that sound?